price-feeder/oracle/provider/binance.go

```diff
@@ -110,7 +110,7 @@ func NewBinanceProvider(
 	provider := &BinanceProvider{
 		wsURL:           wsURL,
 		wsClient:        wsConn,
-		logger:          logger.With().Str("provider", "binance").Logger(),
+		logger:          logger.With().Str("provider", types.ProviderBinance.String()).Logger(),
```

Comment:
```suggestion
		logger:          logger.With().Str("provider", string(types.ProviderBinance)).Logger(),
```

Reply:
this should be more efficient.

Reply:
done at https://github.com/umee-network/umee/pull/1175/commits/5dfae8bf1207f167bb90a5755bc234c41c243264